velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -785,6 +785,16 @@ size_t ParquetReader::numberOfRowGroups() const {
   return readerBase_->fileMetaData().row_groups.size();
 }
 
+thrift::CompressionCodec::type ParquetReader::getCompressionKind(
```

Comment:
We didn't want to expose thrift structs/enums in the reader. Instead, it should be `common::CompressionKind` which is also returned for the DRWF reader. So the mapping from codec to kind has to occur here already.

Reply:
> With the recent changes in the `ThriftCodecToCompressionKind()` function in PageReader.h , an argument can no longer be passed into the `ThriftCodecToCompressionKind()` function, meaning that the compressionKind needs to be extracted in a different way.

That was the plan all along , need to find out another way.

Reply:
Since there now is a re-use, let's move `common::CompressionKind ThriftCodecToCompressionKind` to ParquetReaderUtil.h/ParquetReaderUtil.cpp

Reply:
Maybe I don't remember the conversation, but didn't we decide to not expose the `thrift::CompressionCodec::type` through the reader and instead return `common::CompressionKind`. The function would use a util function that does the codec conversion from ParquetReaderUtil.h/ParquetReaderUtil.cpp.

Reply:
Yes , you are right, fixing that.